pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -803,9 +798,9 @@ func handleLogEvent(event logEntry, reportLogs *logs.LogBundle) bool {
 
 // Returns true if a message was successfully sent
```

Comment:
Please expand the comment to specify the new return

Reply:
will do